app/support/products/scheduling_support.rb

```diff
@@ -98,6 +98,14 @@ def available_schedule_rules(user)
     end
   end
 
+  def online!
+    offline_reservations.current.update_all(reserve_end_at: Time.current)
```

Comment:
Maybe? Reserve times should equal actuals if we do, though at this point nothing is looking at offline actuals. Will it matter with reporting?